src/trajectory.jl

```diff
@@ -246,13 +254,29 @@ function randcat(rng, zs::AbstractVector{<:PhasePoint}, unnorm_ℓP::AbstractMat
 end
 
 function samplecand(rng, τ::StaticTrajectory{MultinomialTS}, h, z)
-    zs = step(τ.integrator, h, z, τ.n_steps; res=[z for _ in 1:abs(τ.n_steps)])
+    n_steps = abs(τ.n_steps)
+    n_fwd = rand(0:n_steps) # FIXME: deal with multi-chain generically
```

Comment:
Yes it's correct.

Reply:
One thing to correct / be explict here: the trajectory has `n_steps + 1` *points* while has a length of `n_steps * step_size`; `zs` is an array with length of `n_steps + 1`.

Reply:
By "length" I meant the _number of steps_ :+1: And (I'm being a bit pedantic here) I feel like using "length" about `n_steps * step_size` is slightly incorrect, since it's _not_ the length of the trajectory in, say, Euclidean distance as you'd normally interpret it, but rather it's the "interval of time" that you integrate, right?

Reply:
Yes, that's integration time if we want to be pedantic.